search/django/weedid/views.py

```diff
@@ -229,9 +229,10 @@ def update_categories(request):
     )
     try:
         updated_weedcoco_json = set_categories(weedcoco_path, categories)
-        validate(updated_weedcoco_json, schema="compatible-coco")
+        validate_json(updated_weedcoco_json, schema="compatible-coco")
     except JsonValidationError as e:
-        return HttpResponseBadRequest(e.message)
+        traceback.print_exc()
+        return HttpResponseBadRequest(json.dumps(e.get_error_details()))
```

Comment:
This should actually be 
```suggestion
        return HttpResponseBadRequest(json.dumps(e.get_error_details()), content_type="application/json")
```
shouldn't it?